filebeat/input/v2/compat/compat.go

```diff
@@ -104,11 +104,13 @@ func (f *factory) Create(
 func (r *runner) String() string { return r.input.Name() }
 
 func (r *runner) Start() {
+	r.wg.Add(1)
 	log := r.log
 	name := r.input.Name()
 
 	go func() {
-		log.Infof("Input %v starting", name)
+		defer r.wg.Done()
+		log.Infof("Input '%s' - '%s' starting", name, r.id)
```

Comment:
The runner is created with a logger that should always have the ID in each statement, does the `With("id"` here not actually work?

https://github.com/elastic/beats/blob/cad3c7f951e55221712b96a2b32461c09c426e27/filebeat/input/v2/compat/compat.go#L94-L96